src/System.Net.Ping/src/System/Net/NetworkInformation/Ping.Unix.cs

```diff
@@ -160,7 +165,7 @@ public partial class Ping
                 throw new PlatformNotSupportedException(SR.net_ping_utility_not_found);
             }
 
-            string processArgs = UnixCommandLinePing.ConstructCommandLine(buffer.Length, address.ToString(), isIpv4);
+            string processArgs = UnixCommandLinePing.ConstructCommandLine(buffer.Length, address.ToString(), isIpv4, options?.Ttl ?? 0);
```

Comment:
Is there a way to test this?

Reply:
test if the option is supported by ping?

Reply:
Sorry, I didn't mean to comment on this line specifically... I meant test that this PR is doing what was intended, that the TTL is being respected now.

Reply:
yes. I did run wireshark when running on my Ubuntu VM. I'll re-run the test on OSX. 
As far as unit test, we would need stable server with distance greater than 1. 
Than we could craft outerloop test and verify results with different TTL.